docs/interactions/Application_Commands.md

```diff
@@ -4,8 +4,9 @@ Application commands are commands that an application can register to Discord. T
 
 ## Application Command Object
 
-> info
-> `CHAT_INPUT` command names must be all lowercase matching `^[\w-]{1,32}$`. `USER` and `MESSAGE` commands may be mixed case and can include spaces.
+###### Application Command Naming
```

Comment:
This subheading format is a bit funky because we tend to use 6 hashes just for table headers, but putting it any more top-level would mean that the rest of the tables would get linked to the naming subheading, which would be weird.

The section could be moved somewhere else, but it makes sense here.

All in all, fine, just thinking out loud.